robot/impl/data/diff_config_1.json

```diff
@@ -3,7 +3,10 @@
         {
             "name": "arm1",
             "type": "arm",
-            "model": "fake"
+            "model": "fake",
+            "attributes": {
+                "model-path": "../../components/arm/fake/fake_model.json"
```

Comment:
not sure if relative path is the way to go

Reply:
Absolute path won't really work here, because that'll be dependent on the filesystem of the person running the test. Relative will always be correct because it's self-contained.

Until someone moves something. In which case they'll see all the failed tests and update.